uportal-war/src/main/webapp/WEB-INF/jsp/BackgroundPreference/viewBackgroundPreference.jsp

```diff
@@ -27,30 +27,25 @@
     <portlet:param name="action" value="savePreferences"/>
 </portlet:actionURL>
 
-<style>
+<style id="#${n}background-preference-stylesheet">
 
     /* Styles to set the background to the previously-selected image */
     <c:if test="${backgroundImage ne null}">
-    ${backgroundContainerSelector}, html.um-dashboard {
+    ${backgroundContainerSelector}::before,
+    html.um-dashboard {
```

Comment:
@asgrant @bpowell  are `um-*` selectors being used in the latest Oakland uMobile client?